packages/protocol/contracts/tko/TaikoToken.sol

```diff
@@ -22,4 +26,10 @@ contract TaikoToken is TaikoTokenBase {
         // Mint 1 billion tokens
         _mint(_recipient, 1_000_000_000 ether);
     }
+
+    function delegates(address account) public view virtual override returns (address) {
+        // Special checks to avoid reading from storage slots
+        if (account == _TAIKO_L1 || account == _ERC20_VAULT) return address(0);
+        else return super.delegates(account);
+    }
```

Comment:
Should not we apply it on L2's (`BridgedTaikoToken`) ERC20Vault too ? I know gas is cheaper there.. just for the sake of synchronicity.

Reply:
I think it's not necessary.